content/sql/concepts/commands/terms/top/top.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'TOP'
```

Comment:
I'm wondering if this entry should be repurposed as a concept entry and renamed as 'SELECT TOP' (similar to the existing [SELECT DISTINCT](https://www.codecademy.com/resources/docs/sql/commands/select-distinct) entry).

cc: @SSwiniarski